fastly/resource_fastly_service_v1_test.go

```diff
@@ -614,6 +614,18 @@ func TestAccFastlyServiceV1_defaultHost(t *testing.T) {
 						"fastly_service_v1.foo", "default_ttl", "3400"),
 					resource.TestCheckResourceAttr(
 						"fastly_service_v1.foo", "stale_if_error_ttl", "43200"),
+						"fastly_service_v1.foo", "default_host", defaultHost),
```

Comment:
@bridgetlane looks like a bad merge. I think you need an extra `resource.TestCheckResourceAttr(` added above this line.

Reply:
that should hopefully fix all the other lint errors as I think they're all falling out from this first syntax error.